src/renderer/containers/Onboard/CreatePassword/validation.ts

```diff
@@ -0,0 +1,27 @@
+import {ObjectSchema} from "@hapi/joi";
+import {Joi} from "../../../services/validation";
+
+export const passwordFormSchema = Joi.object().keys({
+    password: Joi.password()
+        .min(6)
+        .max(12)
+        .numOfLower(1)
+        .numOfUpper(1)
+        .numOfNumbers(1)
+        .numOfSigns(1)
```

Comment:
You could solve it all with this regex: 
`"^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{6,12}$"`

Reply:
but leave it as is.